Zend/zend_dtrace.c

```diff
@@ -68,20 +68,22 @@ ZEND_API void dtrace_execute_ex(zend_execute_data *execute_data)
 	if (DTRACE_FUNCTION_ENTRY_ENABLED() || DTRACE_FUNCTION_RETURN_ENABLED()) {
 		classname = get_active_class_name(&scope);
 		funcname = get_active_function_name();
+		prev_funcname = get_prev_active_function_name();
+		prev_classname = get_prev_active_class_name(&prev_scope);
```

Comment:
but if you do not care about `prev_scope`, if I understand your own implementation, you can just pass NULL maybe ?

Reply:
sure, done